packages/web-components/fast-components/src/toolbar/index.ts

```diff
@@ -29,8 +29,8 @@ export class Toolbar extends FoundationToolbar {
 }
 
 /**
- * The FAST toolbar Custom Element. Implements {@link @microsoft/fast-foundation#Toolbar},
- * {@link @microsoft/fast-foundation#ToolbarTemplate}
+ * A function that returns a {@link @microsoft/fast-foundation#Toolbar} registration for configuring the component with a DesignSystem.
+ * Implements {@link @microsoft/fast-foundation#ToolbarTemplate}
```

Comment:
For consistency, below it should probably say "Generates HTML Element: \<fast-toolbar\>"